Source/x11/XGServerWindow.m

```diff
@@ -3072,10 +3104,11 @@ - (void) restrictWindow: (int)win toImage: (NSImage*)image
         {
 	    if ([rep samplesPerPixel] == 4)
 	      {
-		pixmap = xgps_cursor_mask(dpy, GET_XDRAWABLE(window),
-					  [rep bitmapData],
-					  [rep pixelsWide], [rep pixelsHigh],
-					  [rep samplesPerPixel]);
+		pixmap = image_mask(dpy, GET_XDRAWABLE(window),
+                                    [rep bitmapData],
+                                    [rep pixelsWide], [rep pixelsHigh],
+                                    [rep samplesPerPixel],
+                                    ALPHA_THRESHOLD);
```

Comment:
What is the value of ALPHA_THRESHOLD at this point?

Reply:
#define is at line 3080.